gaffer-as-a-service/gaas-rest/pom.xml

```diff
@@ -295,5 +296,63 @@
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>docker-build</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <properties>
+                <buildType>buildTar</buildType>
```

Comment:
Didn't realise you could put properties here too. I'm guessing that you can alter them from the command line?

Reply:
Sure. Command line properties value overwrites.